modules/homestakeros/default.nix

```diff
@@ -185,6 +186,54 @@ in
         }
       )
 
+      #################################################################### SSV
+      # cfg: https://docs.ssv.network/run-a-node/operator-node/installation#create-configuration-file
+      (
```

Comment:
This could have a note that it's done this way because of challenges of using env variables.

Reply:
I opened an issue at https://github.com/bloxapp/ssv/issues/1138 and added a reference.